test/markdownlint-test.js

```diff
@@ -1037,6 +1037,35 @@ tape("configMultiple", (test) => {
     });
 });
 
+tape("configMultipleWithRequireResolve", (test) => {
+  test.plan(2);
+  const fakeNodeModulesDir = path.resolve(__dirname, "./node_modules");
+  try {
+    fs.mkdirSync(fakeNodeModulesDir);
+    fs.symlinkSync(
+      path.resolve(__dirname, "./config/pseudo-package"),
+      path.resolve(fakeNodeModulesDir, "test"),
```

Comment:
Could we have the directory name match this test name? One day, maybe there will be another.

Reply:
Do you mean renaming `./config/pseudo-package`, `test` or both? Happy to do so, just need a bit more detail. The test is called `configMultipleWithRequireResolve` (camel-case), which we can’t reuse in dir names without changing.

Reply:
Sorry, I was thinking something like:  fakeNodeModule/configMultipleWithRequireResolve

Reply:
`test/configs/fakeNodeModule/configMultipleWithRequireResolve/package.json` plus `test/configs/fakeNodeModule/configMultipleWithRequireResolve/config.json` ?

or

`test/configs/fakeNodeModule/package.json` plus `test/configs/fakeNodeModule/configMultipleWithRequireResolve.json`?

Sorry got confused a bit. 

[If we look at Prettier](https://github.com/prettier/prettier/tree/master/tests_integration/plugins), they just store `node_modules/whatever` right in their repo, so that they don't need to symlink the files. We could to something similar by moving `test/configs/pseudo-package` to `test/node_modules/pseudo-package` or whatever name that would closer match the test name. I would not use `configMultipleWithRequireResolve` because it's camelCase, which is [not allowed for npm package names](https://docs.npmjs.com/files/package.json#name).

Reply:
Happy for you to push into this PR if that's less time-consuming 🙂

Reply:
test/node_modules/pseudo-package sounds great! Agree with adding an exclude line to .gitignore for test/node_modules. Or neither and let me tweak a bit when I come back to this after the patch release. 😀

Reply:
Done! 👍

Reply:
Love it! No copy, no clean-up.